Notre Dame Philosophical Reviews.js

```diff
@@ -0,0 +1,207 @@
+{
+  "translatorID": "a823550b-6475-4b20-8539-a3c416906228",
+  "label": "Notre Dame Philosophical Reviews",
+  "creator": "Emiliano Heyns",
+  "target": "^https://ndpr\\.nd\\.edu",
```

Comment:
put a slash at the end of this -- I don't understand this 100%, but in case people use this with a proxy it'll make sure the regex still works.